src/en/authors-charm-components.md

```diff
@@ -9,7 +9,7 @@ metadata, configuration data, and hooks with some extra support files.
 
 A charm requires only a single file in order to be considered valid by juju:
 
- - `metadata.yaml` [describes the charm](./authors-charm-metadata.html) and the
+ - `metadata.yaml` [describes the charm](../authors-charm-metadata.html) and the
```

Comment:
I don't think this is the path we want, it should be just
      [describes the charm](authors-charm-metadata.md)
or
     [describes the charm](./authors-charm-metadata.md)

Reply:
Good point, I'll revert the html links and leave the image links.